src/parser/parsing_table.cpp

```diff
@@ -17,6 +17,13 @@ void ParsingTable::BuildTable() {
   for (auto &nt : non_terminals_) {
     if (follows_.count(nt) != 0U) {
       for (const auto &symbol : follows_[nt]) {
+        if (table_[nt][symbol] != std::string(ERROR_TOKEN)) {
+          std::string ret;
+          ret += "Error duplicate entry in parsing table, ";
```

Comment:
"error: duplicate entry ..."
First character is not capitalized in unix systems generally.
no space before ":"
should be "production: " and not "Production : " etc.